features/cellular/framework/targets/UBLOX/PPP/UBLOX_PPP_CellularPower.h

```diff
@@ -35,6 +35,11 @@ class UBLOX_LISA_U_CellularPower : public AT_CellularPower
     virtual nsapi_error_t off();
 };
 
+class UBLOX_LISA_U_CellularPower  : public UBLOX_PPP_CellularPower
+{
+    MBED_DEPRECATED_SINCE("mbed-os-5.9", "This API will be deprecated, Use UBLOX_PPP_CellularPower instead of UBLOX_LISA_U_CellularPower.");
```

Comment:
Hello, this isn't the correct way to mark a class as deprecated.

The deprecated attribute needs to be before the class:
``` cpp
MBED_DEPRECATED_SINCE("mbed-os-5.9", "This API will be deprecated, Use UBLOX_PPP_CellularPower instead of UBLOX_LISA_U_CellularPower.")
class UBLOX_LISA_U_CellularPower  : public UBLOX_PPP_CellularPower
{
   blablabla
```

Otherwise this won't compile on GCC or ARM.
https://github.com/ARMmbed/mbed-os/blob/master/platform/mbed_toolchain.h#L297

Reply:
Actually, I put a semi colon at the end of the deprecated warning and it compiled successfully.

Thanks for the correction. I have modified the code now.

Reply:
Oh that's interesting. Although the deprecated message still won't show up.

Thanks for fixing this :+1: